File: backend/src/models/position.ts
class Position {
  public type: string;
  public coordinates: number[];
  public createdAt: string;

  constructor() {
    this.type = "Point";
    this.coordinates = [0.0, 0.0];
    this.createdAt = new Date().toISOString();
  }
}

export default Position;

File: webapi-spring/src/main/java/com/monitorz/webapi/data/Position.java
package com.monitorz.webapi.data;

import java.util.List;

public class Position {
    List coordinates;
    String type = "Point", createdAt;
    int  index;

    public List getCoordinates() {
        return coordinates;
    }

    public void setCoordinates(List coordinates) {
        this.coordinates = coordinates;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }
}
